packages/bitcore-wallet-client/src/lib/common/constants.ts

```diff
@@ -55,16 +55,13 @@ export const Constants = {
   ],
 
   BITPAY_SUPPORTED_MATIC_ERC20: [
-    'eth_m',
     'usdc_m',
-    'pax_m',
-    'gusd_m',
     'busd_m',
     'dai_m',
     'wbtc_m',
     'shib_m',
     'ape_m',
-    'euroc_m'
+    'matic_m'
```

Comment:
I guess this is wrapped MATIC?

Reply:
MATIC Token is its name

Reply:
In that case I don't think this is needed. MATIC is the base asset on the Polygon chain similar to how ETH is the base asset on the Ethereum chain. We don't need to add it as a token here.

Reply:
in the pr description image is there a list of all the tokens that we support, `matic_m` is simply a token in the polygon chain called "MATIC Token" different from the MATIC coin that would be the equivalent of ETH in its own chain